scripts/docs-generation/3rdPartyDocs.js

```diff
@@ -0,0 +1,109 @@
+const fs = require('fs')
+const path = require('path')
+const request = require('request')
+
+const { buildPreface } = require('../utils/helpers')
+const reporters3rdParty = require('./3rd-party/reporters.json')
+const services3rdParty = require('./3rd-party/services.json')
+
+const packages3rdParty = [...reporters3rdParty, ...services3rdParty]
+const plugins = {
+    reporter: ['Reporter', 'Reporter'],
+    service: ['Services', 'Service']
+}
+const githubHost = 'https://github.com/'
+const githubRawHost = 'https://raw.githubusercontent.com/'
+const githubReadme = 'master/README.md'
+
+const readmeHeaderLines = 9
+const readmeHeaders = ['===', '# ']
+const readmeBadges = 'https://badge'
+
+/**
+ * Generate docs for 3rd party reporters and services
+ * @param {object} sidebars website/sidebars
+ */
+exports.generate3rdPartyDocs = async (sidebars) => {
+    for (const [, [namePlural, nameSingular]] of Object.entries(plugins)) {
+        for (const { packageName, title, githubUrl } of packages3rdParty) {
+            const readme = await downloadReadme(githubUrl)
+            const id = `3rd-party-${packageName}`.replace(/@/g, '').replace(/\//g, '-')
```

Comment:
```suggestion
            const id = `${packageName}`.replace(/@/g, '').replace(/\//g, '-')
```

I don't think we need to specify this. Maybe better to attach a paragraph at the end or beginning saying that this is a 3rd party script and issues with it should be posted in the project repo of the 3rd party